src/stores/index.ts

```diff
@@ -10,6 +11,7 @@ import {
 export type RootState = {
 	chat: ReturnType< typeof chatReducer >;
 	wordpressVersions: ReturnType< typeof wordpressVersionsReducer >;
+	[ wpComApi.reducerPath ]: ReturnType< typeof wpComApi.reducer >;
```

Comment:
Let's be consistent with the other reducers. I think we are mostly creating complexity by making this value dynamic instead of static, but if you feel strongly about it, then let's also change it for the other reducers.